server/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesResponseTests.java

```diff
@@ -118,6 +123,35 @@ public void testFailureSerialization() throws IOException {
         }
     }
 
+    /**
+     * check that failure serialization between different minor versions after 7.13 works
+     */
+    public void testMixedVersionFailureSerialization() throws IOException {
+        FieldCapabilitiesResponse original = createResponseWithFailures();
+        FieldCapabilitiesResponse deserialized;
+        try (BytesStreamOutput output = new BytesStreamOutput()) {
+            Version outVersion = VersionUtils.randomVersionBetween(random(), Version.V_7_13_0, Version.CURRENT);
```

Comment:
do we already test elsewhere what happens when the version is before 7.13? I mean testing that the failure is silently ignored and nothing breaks reading from the wire.